wikia/common/logger/wikia_json_logs_formatter.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
```

Comment:
That filename is just too long and `wikia` is redundant. Also this module, apart form the formatter, also has a method for configuring json logger. `json_logger` as a name should be fine.